src/twisted/internet/test/connectionmixins.py

```diff
@@ -127,29 +127,28 @@ def buildProtocol(self, addr):
         return self._protocol
 
 
-def runProtocolsWithReactor(
-    reactorBuilder, serverProtocol, clientProtocol, endpointCreator
+def runProtocolsWithReactorInstance(
+    reactor, reactorBuilder, serverProtocol, clientProtocol, endpointCreator
 ):
     """
-    Connect two protocols using endpoints and a new reactor instance.
+    Connect two protocols using endpoints and an existing reactor instance.
 
     A new reactor will be created and run, with the client and server protocol
```

Comment:
This docstring seems to be wrong now; no new reactor is created in this function.